lib/run_loop/core.rb

```diff
@@ -68,7 +68,8 @@ def self.run_with_options(options)
       instruments = options[:instruments] || RunLoop::Instruments.new
 
       # Find the Device under test, the App under test, UIA strategy, and reset options
-      device = RunLoop::Device.detect_device(options, xcode, simctl, instruments)
+      # Experimental selected device detection
+      device = RunLoop::DetectAUT::Xcode.detect_selected_device || RunLoop::Device.detect_device(options, xcode, simctl, instruments)
```

Comment:
> TRY_TO_FIND=1

We do not want to introduce any new environment variables.

* Hard to document
* New users don't understand
* Almost impossible to deprecate - look at how people still use OS and DEVICE=device